mars-cli/tests/test_ftp_upload.py

```diff
@@ -13,10 +13,17 @@ def test_upload_login_failure():
         uploader.upload([Path("./tests/fixtures/not_a_json_file.txt")])
 
 
-@pytest.mark.skip(reason="Relies on real ENA credentials in test_credentials_example.json")
+@pytest.mark.skip(
+    reason="Relies on real ENA credentials in test_credentials_example.json"
+)
 def test_upload_success():
     # For local testing, add ENA username/password to test_credentials_example.json
     with open("./tests/test_credentials_example.json") as f:
         creds = json.load(f)
     uploader = FTPUploader("webin2.ebi.ac.uk", creds["username"], creds["password"])
-    uploader.upload([Path("../test-data/ENA_TEST2.R1.fastq.gz"), Path("./tests/fixtures/not_a_json_file.txt")])
+    uploader.upload(
+        [
+            Path("../test-data/ENA_TEST2.R1.fastq.gz"),
+            Path("./tests/fixtures/not_a_json_file.txt"),
+        ]
+    )
```

Comment:
also thanks for fixing all my style issues, I keep forgetting to run the checks locally :sweat_smile: